android/src/main/java/io/mosip/tuvali/openid4vpble/Openid4vpBleModule.kt

```diff
@@ -39,6 +40,11 @@ class Openid4vpBleModule(private val reactContext: ReactApplicationContext) :
     return NAME
   }
 
+  @ReactMethod(isBlockingSynchronousMethod = true)
```

Comment:
This annotation may not be needed as it should be private method to Tuvali repo

Reply:
we are calling setTuvaliVersion() from index.tsx and to expose kotlin methods to js layer we have to annotate it with "@ReactMethod " annotation and if we don't add annotation we are not able to access it in index.tsx. But we tried searching for ways how to use this function only within our project and exclude it when adding tuvali as dependency in inji and tried different annotation @hide and RestrictTo and also made method as private but still we are able to access it in inji.